docs/DIAGNOSTIC.md

```
@@ -0,0 +1,101 @@
+      "span": null,
+      "suggestion": "'[LABEL name=good_end]' を追加するか、ジャンプ先を修正してください"
+    }
+  ]
+}
+```
+
+---
+
+## 5. CLIでの人間向け表示
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)